packages/@storefront/flux-capacitor/src/flux-capacitor.ts

```diff
@@ -52,12 +73,74 @@ class FluxCapacitor extends Emitter {
     delete this.__config;
   }
 
+  initHistory({ build, parse, initialUrl, pushState, replaceState }: History) {
+    this.history = {
+      build,
+      initialUrl,
+      pushState,
+      replaceState,
+      noop: () => null,
+      parse: (u) => Promise
+        .resolve(parse(u))
+        .catch((e) => console.warn('UrlService parse promise failed', e)),
```

Comment:
Need to replace this w/ `app.log` somehow

Reply:
I think we just use `console` everywhere in flux. The larger ticket would be to move the logger into flux.

Reply:
there's no other `console` in flux I just searched for it

Reply:
Actually, that makes sense because we don't assume our environment in flux. So have we been silently catching errors?

But also, I think this function now knows too much about its caller since it refers to `UrlService`. Is it even useful to catch here? I think it should be up to the caller to handle because a following `then` is meaningless after getting to the `catch`.